### ISSUE TITLE
Top content mode

### ISSUE THREAD

**Issue by piotrros** [huynguyencong/ImageScrollView#46]
```
@objc public enum ScaleMode: Int {
        case aspectFill
        case aspectFit
        case widthFill
        case heightFill
}
```

Only these modes are supported, but how can I use something like "ContentMode.top" so it'll fit the image, but instead of centering it, it'd move the image to the top? I've tried limiting view's height by manually setting up aspect ratio constraint, but then it causes the other issue, which I've described here:

https://github.com/huynguyencong/ImageScrollView/issues/45

**Comment by huynguyencong** [comment]
Sorry, but I can't help. This library just support those scale mode. Please fork it and implement it by yourself.

**Comment by piotrros** [comment]
Sure, I've resolved issue #45 and because of this I can just make a container the right size, so it can be aspect fit.
